Vic2ToHoI4/Source/HOI4World/Localisations/AllReplacementRules.cpp

```diff
@@ -0,0 +1,28 @@
+#include "AllReplacementRules.h"
+
+
+
+HoI4::AllReplacementRules::AllReplacementRules(std::istream& theStream)
+{
+	registerKeyword(std::regex("[a-zA-Z_]+"), [this](const std::string& language, std::istream& theStream)
+	{
+		LanguageReplacementRules theRules(theStream);
+		rulesInLanguages.insert(std::make_pair(language, theRules));
+	});
+
```

Comment:
I try to always wrap up with:
	registerKeyword(std::regex("[a-zA-Z0-9_\\.:]+"), commonItems::ignoreItem);
Even when you already catch mostly anything - if only to maintain uniformness across all parsers. Can ignore this if you wish.

Reply:
Forgot about those, thanks!